scalahost/sbt/src/sbt-test/sbt-scalahost/semantic-example/test

```diff
@@ -0,0 +1,11 @@
+> app/testOnly Fat
+> set every scalametaSemanticdb := ScalametaSemanticdb.Slim
+> clean
+> app/testOnly Slim
+> clean
+> set every scalametaSemanticdb := ScalametaSemanticdb.Fat
```

Comment:
I need every here since I want to define it for all projects. When I tried without `every` it only changed the setting in the root project.